spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/security/BasicAuthSecurityConfiguration.java

```diff
@@ -157,6 +161,9 @@ protected void configure(HttpSecurity http) throws Exception {
 		http.addFilterBefore(sessionRepositoryFilter,
 				ChannelProcessingFilter.class).csrf().disable();
 		http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.IF_REQUIRED);
+
+		securityStateBean.setAuthenticationEnabled(true);
+		securityStateBean.setAuthorizationEnabled(true);
```

Comment:
It'd be nice to get comment from @rwinch about a usage of `securityStateBean`. It'd be nice if that bean could be created once with correct settings from a getgo if it's intended to only get passed to security info page.